index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Richard Rabulan | Web Developer</title>
+    <link rel= "stylesheet" type="text/css" href= "normalize.css">
+    <link rel= "stylesheet" type="text/css" href= "style.css">
+</head>
+<a name="top"></a>
+<body>
+    <header class="navBar featureImage">
+        <div class="container">
+                <ul>
+                    <li><a href="#top" class="navDefault">HOME</a></li>
+                    <li><a href="#about">ABOUT ME</a></li>
+                    <li><a href="#feature">PORTFOLIO</a></li>
+                    <li><a href="#footer">CONTACT</a></li>
+                    <li><a href="#top" class="navHidden">HOME</a></li>
+                </ul>
+            <h1>RICHARD RABULAN</h1>
+            <p>Web Developer</p>
+            <a href="#feature" class="more">MORE</a>
+        </div>
+    </header>
+    <section class="portfolio">
+        <a id="feature" name="feature"></a>
+        <div class="boxes">
```

Comment:
could you put the class of boxes on the section instead so you can get rid of extra div?